docs/xml/catalog-xmldata.xml

```diff
@@ -34,14 +34,13 @@
 			quite large.
 		</para>
 
-		<important>
+		<warning>
```

Comment:
Deprecation into is usually an `important` message box in the spec, not a `warning`. Important messages are also orange, so really hard to miss, while `warning` boxes are rendered bright red and usually in the spec are used to highlight some common pitfall or trait in the active, non-legacy spec that users must be aware of or pay special attention to (while "don't use legacy things" isn't one of those).

Reply:
Yes, you're right.